lib/rubocop/cop/style/if_unless_modifier.rb

```diff
@@ -35,13 +35,15 @@ def autocorrect(node)
           cond, body, _else = if_node_parts(node)
 
           oneline =
-            "#{body.loc.expression.source} #{node.loc.keyword.source} " +
-            cond.loc.expression.source
+            (
+              "#{body.loc.expression.source} #{node.loc.keyword.source} " +
+              cond.loc.expression.source
+            ).dup
           first_line_comment = processed_source.comments.find do |c|
             c.loc.line == node.loc.line
           end
           if first_line_comment
-            oneline << ' ' << first_line_comment.loc.expression.source
+            oneline << ' '.dup << first_line_comment.loc.expression.source
```

Comment:
I think string concatenation is slow, so I use << as much as possible in [this commit](https://github.com/sachin21/rubocop/commit/5d9cc8d2310b0c3733d65d75e7b3ac5d69b96ac1#diff-04f16a5074ec69d85f7f5612d212824aR46).

ref: http://greyblake.com/blog/2012/09/02/ruby-perfomance-tricks/